docs/src/gensim_theme/static/doctools.js

```diff
@@ -1,247 +0,0 @@
-/*
```

Comment:
suspicious, why this removed (same question for all `*.js` files)

Reply:
Because I moved the whole thing to the `../`? :)

Reply:
this **should** be moved (but this removed), check your repository please